common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOtherCache.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.identity.common.internal.cache;
+
+import android.content.Context;
+import androidx.annotation.VisibleForTesting;
+import com.microsoft.identity.common.adal.internal.cache.IStorageHelper;
+import com.microsoft.identity.common.adal.internal.cache.StorageHelper;
+
+public class MsalOtherCache {

```

Comment:
Maybe a more meaningful name? `DefaultAuthorityValidationMetadataCache` as a thought? Maybe `SharedPreferencesAuthorityValidationMetadataCache`?

Thinking out loud.... I might suggest adding an abstract superclass (or interface) that defines its methods but _doesn't_ specify a storage mediuem (like `SharedPreferences`) so that if you in the future want to change from `SharedPreferences` to something else you can

Reply:
I changed it to interface IDefaultAuthorityValidationMetadataCache